src/core/lib/gprpp/inlined_vector.h

```diff
@@ -222,12 +233,21 @@ class InlinedVector {
     }
   }
 
+#if 1
   typename std::aligned_storage<sizeof(T)>::type inline_[N];
+#else
```

Comment:
Again, please add a TODO with an issue.

Reply:
This is removed because it will be removed anyway.